server/graphql/v1/mutations.js

```diff
@@ -478,64 +473,6 @@ const mutations = {
       return addFundsToCollective(args.order, req.remoteUser);
     },
   },
-  createUpdate: {
```

Comment:
It's fantastic that you implemented these mutations in V2, but we should not remove them from V1 directly - otherwise the people running an "old" version of the frontend (because we haven't deployed it before the API update, or because they haven't refreshed their browser yet) will get a crash when using them.

The way to go with this kind of changes is:
1. Implement the new mutation on V2
2. Mark the one on V1 as deprecated with `deprecationReason` 
3. Deploy everything
4. Remove mutation from V1 after a few days